src/test/java/spoon/test/annotation/AnnotationTest.java

```diff
@@ -299,7 +299,7 @@ public void testAnnotatedElementTypes() {
 		// check annotated element type of the package annotation
 		List<CtAnnotation<?>> annotations = pkg.getAnnotations();
 		assertEquals(2, annotations.size());
-		assertTrue(annotations.get(0).getAnnotatedElement().equals(pkg));
+		assertEquals(annotations.get(0).getAnnotatedElement(), pkg);
```

Comment:
the expected value should be first.

Reply:
@monperrus 
Thanks! I fixed this one.

Could you check other tests I modified?